src/components/Card.js

```diff
@@ -1,47 +1,91 @@
-class Card {
-  constructor({ name, link }, cardTemplateSelector, handleCardClick) {
+export default class Card {
+  constructor(
+    { name, link, _id, isLiked },
+    cardTemplateSelector,
+    handleCardClick,
+    handleDeleteClick,
+    handleLikeClick
+  ) {
     this._name = name;
     this._link = link;
+    this._id = _id;
+    this._isLiked = isLiked;
     this._handleCardClick = handleCardClick;
+    this._handleDeleteClick = handleDeleteClick;
+    this._handleLikeClick = handleLikeClick;
     this.cardTemplate =
       document.querySelector(cardTemplateSelector).content.firstElementChild;
+    this._modal = document.querySelector("#card-delete-modal");
+    this._handleEscClose = this._handleEscClose.bind(this); /
```

Comment:
`this._handleEscClose = this._handleEscClose.bind(this)` is not needed in `Card`